lib/draw.dart

```diff
@@ -5,7 +5,7 @@
 
 library draw;
 
-export 'package:draw/src/reddit.dart';
+export 'package:draw/src/reddit.dart' hide RedditBaseInitializedMixin;
```

Comment:
Would you mind confirming:
I looked into the Dart Docs (https://www.dartlang.org/guides/language/language-tour), there was no reference or use of `hide` for exporting classes. What does it do?
(Primary reason for asking is I noticed you removed a few public comments in "base_impl.dart")

Reply:
See docs for `hide` and `show` [here](https://www.dartlang.org/guides/language/language-tour#importing-only-part-of-a-library).